specs/ExtendedAcceleratorKeyPressed.md

```diff
@@ -0,0 +1,187 @@
+# Background
+WebView2 has `ICoreWebView2AcceleratorKeyPressedEventArgs` for the 
+`CoreWebView2Controller.AcceleratorKeyPressed` event. We have been asked to extend this API so
+that developers can indicate that theywould like an existing browser accelerator key to be 
+disabled as a browser accelerator.
+
+In this document we describe the updated API. We'd appreciate your feedback.
+
+# Description
+We propose extending the `ICoreWebView2AcceleratorKeyPressedEventArgs` to allow developers
+to enable/disable the browser from handling accelerator keys with `AllowBrowserAcceleratorHandling` 
+property. 

```

Comment:
Confusing word order because it sounds like it's saying "This lets you control whether the browser handles accelerator keys that have the `AllowBrowserAcceleratorHandling` property. (But no effect on accelerator keys that don't have the `AllowBrowserAcceleratorHandling` property.)" But really it's

> We propose extending the `ICoreWebView2AcceleratorKeyPressedEventArgs` with a new `AllowBrowserAcceleratorHandling` property to allow developers to control whether the browser handles accelerator keys.

Reply:
updated!